crates/goose/src/providers/test_utils.rs

```diff
@@ -0,0 +1,54 @@
+use anyhow::Result;
+use httpmock::MockServer;
+use serde_json::Value;
+use std::fs;
+use std::path::PathBuf;
+
+pub const TEST_FIXTURES_DIR: &str = "tests/fixtures";
```

Comment:
do we expect to see files in this directory, yet?

Reply:
great catch, I missed checking in, I removed it for now but we should look into a separate pr